src/renderers/HeatmapCellRenderer.ts

```diff
@@ -0,0 +1,73 @@
+import ICellRendererFactory from './ICellRendererFactory';
+import MultiValueColumn from '../model/MultiValueColumn';
+import {IDOMRenderContext, ICanvasRenderContext} from './RendererContexts';
+import {ISVGCellRenderer, IHTMLCellRenderer} from './IDOMCellRenderers';
+import {IDataRow} from '../provider/ADataProvider';
+import ICanvasCellRenderer from './ICanvasCellRenderer';
+import {select as d3select} from 'd3';
+
+
+export default class HeatmapCellRenderer implements ICellRendererFactory {
```

Comment:
since this type will be registered as `multiValue` the class and file name should be named the same